Example/Podfile.lock

```diff
@@ -1,7 +1,7 @@
 PODS:
   - ILGDynamicObjC (0.1.0):
-    - ILGDynamicObjC/ILGClasses
-    - ILGDynamicObjC/ILGSwizzler
+    - ILGDynamicObjC/ILGClasses (= 0.1.0)
+    - ILGDynamicObjC/ILGSwizzler (= 0.1.0)
```

Comment:
Weird, theoretically yes. Even if i run `pod update ILGDynamicObjC` specifically, it doesn't actually update.